modules/402-ingress-nginx/template_tests/controller_test.go

```diff
@@ -18,15 +18,27 @@ package template_tests
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+	"sigs.k8s.io/yaml"
 
 	. "github.com/deckhouse/deckhouse/testing/helm"
 )
 
+var golden bool
+
+func init() {
+	flag.BoolVar(&golden, "golden", false, "Generate golden files")
+}
```

Comment:
```suggestion
func init() {
	flag.BoolVar(&golden, "golden", false, "Generate golden files")
}
```
We usually use env var instead of fags.

Reply:
Done